### ISSUE TITLE
Trying to make a docker image with jdbc on 6.0.0

### ISSUE THREAD

**Issue by mdespriee** [confluentinc/cp-docker-images#877]
I'm trying to build a docker image to v6 as following:
```
FROM confluentinc/cp-kafka-connect-base:6.0.0

RUN confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:latest 

ENV MYSQL_DRIVER_VERSION 8.0.21

RUN mkdir -p /usr/share/java/kafka-connect-jdbc
RUN curl https://repo1.maven.org/maven2/mysql/mysql-connector-java/${MYSQL_DRIVER_VERSION}/mysql-connector-java-${MYSQL_DRIVER_VERSION}.jar \
    -o /usr/share/java/kafka-connect-jdbc/mysql-connector-java-${MYSQL_DRIVER_VERSION}.jar
```

Adding a jdbc/mysql connector fails with ```org.apache.kafka.connect.errors.ConnectException: Failed to find any class that implements Connector and which name matches io.confluent.connect.jdbc.JdbcSourceConnector,```

What's wrong ?

**Comment by rmoff** [comment]
Have a look at https://github.com/confluentinc/demo-scene/blob/master/kafka-connect-zero-to-hero/docker-compose.yml#L97-L111 - this is a working example of extending the `base` image. It could well be the `CONNECT_PLUGIN_PATH` is missing from your `Dockerfile` which is causing the problem.

**Comment by mdespriee** [comment]
It seems the right path to put the mysql jar file is `/usr/share/confluent-hub-components/confluentinc-kafka-connect-jdbc/lib/`
Not sure to understand why my version didn't work, but here we go.
Also added `ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"` before jdbc plugin install

**Comment by mdespriee** [comment]
Ok, I managed to make it work. 2 things were wrong.
First, CONNECT_PLUGIN_PATH was missing as you mentioned.
```
ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"
```

Second, the jdbc plugin was not able to load the jar from /usr/share/java/kafka-connect-jdbc. The pluging doc is very misleading on that point.
I made it work by putting the jar with other pre-packaged jars in `/usr/share/confluent-hub-components/confluentinc-kafka-connect-jdbc/lib`

**Comment by rmoff** [comment]
Glad it worked. Can you link me to the doc that you found misleading so that we can fix it? 
Confluent Hub installs to `/usr/share/confluent-hub-components` by default (you can override it if you want) which is why both these things were required.

**Comment by mdespriee** [comment]
* The whole plugin doc (https://docs.confluent.io/kafka-connect-jdbc/current/index.html#mysql-server) mentions `share/java/kafka-connect-jdbc` every paragraph. That's mainly the reason why I downloaded the jar at that place. 
* My docker image was previously based on `confluentinc/cp-kafka-connect:5.5.0`. And the mysql jdbc driver was installed in `/usr/share/java/kafka-connect-jdbc` and worked properly
* To bump to 6.0, I understood that I had to switch to `cp-kafka-connect-base` instead. I don't remember having read any clear doc about the possible deprecation or change of spec.
* When switching to cp-kafka-connect-base, it's the plugin (not the jdbc driver) that was not working at the beginning because of the missing `CONNECT_PLUGIN_PATH`. I think, more than once, I mixed this problem with the "driver not found" problem.
* Then, when the plugin was ok, it's the jdbc jar that was not on the classpath. There are certainly several places where the jar could be copied to, but not `/usr/share/java/kafka-connect-jdbc` any more.
